cmd/eval.go

```diff
@@ -436,6 +447,11 @@ func evalOnce(ctx context.Context, ectx *evalContext) pr.Output {
 	}
 
 	result.Errors = pr.NewOutputErrors(resultErr)
+	if ectx.builtInErrorList != nil {
+		for _, err := range *(ectx.builtInErrorList) {
+			result.Errors = append(result.Errors, pr.NewOutputErrors(&err)...)
```

Comment:
Let's throw in an `err := err` before this. It should resolve the location problem.

Reply:
Hey, I'm not sure I follow you here. Are you referring to the error source code location or something else?

Reply:
Fixed in https://github.com/open-policy-agent/opa/pull/5550